Pika/Views/EyedropperItem.swift

```diff
@@ -35,12 +35,7 @@ struct EyedropperItem: View {
                 .toast(
                     isShowing: $showToast,
                     color: eyedropper.color.getUIColor(),
-                    text: Text(String(
-                        format: NSLocalizedString("color.copy.toast", comment: "Copied foreground / background"),
-                        eyedropper.type == .foreground
-                            ? NSLocalizedString("color.foreground", comment: "Foreground").lowercased()
-                            : NSLocalizedString("color.background", comment: "Background").lowercased()
-                    ))
+                    text: Text(String(NSLocalizedString("color.copy.toast", comment: "Copied foreground / background")))
```

Comment:
It'd be great if this could still show "Copied foreground" or "Copied background", @AClmbr—would the best way to achiveve that be to add something like the following to `eyedroppers.swift` / to the localized strings?

```swift
var description: String {
    switch self {
        case .foreground: return NSLocalizedString("color.copy.toast.foreground", comment: "Copied foreground")
        case .background: return NSLocalizedString("color.copy.toast.background", comment: "Copied background")
    }
}
```

Reply:
As you might have seen, I edited the localized value of the key `color.copy.toast` from `Copied %@` to `Copied` in English. Let me explain why.

- The main reason was that in English the adjective comes before, so `Copied %@` with `NSLocalizedString("...", comment: "...").lowercased()` (the previous code) becomes `Copied foreground` for instance, but in French, the adjective comes after -> `premier plan copié`, without the capital letter at the beginning. So in order to preserve it, I had to change the translation both in French and in English,
- The other reason is that the toast is displayed in a specific view, so you already know which "plan" you are copying. If the toast appears at the left, you know that the foreground has been copied and if it is at right, the background. It saves some space in the UI and removes a repetition.

But we can definitely edit this if you want. My guess is that you'll want to display "Copied red" (the translated name of the current color), isn't it?

Reply:
Yeah, you make a great point, @AClmbr. Thinking more about this, I'm not sure needing more information than "copied" is valuable or necessary, given they're displayed within the swatch itself. How about the following, simplified wording?

```swift
text: Text(String(NSLocalizedString("color.copy.toast", comment: "Copied")))
```

Reply:
Oh lol, apologies @AClmbr, I just reread your point and realised I phrased agreeing with you as a question as I'd missed your point about the update to the `color.copy.toast`. 😅  I think the only update would be to the line of code I used above, to match the comment to what is being output. But totally agreed.

Reply:
I will change it tomorrow 👌

Reply:
Awesome. The only other bit I picked up on was using the new eyedropper in `PikaTouchBar.swift`:

```diff
--- a/Pika/TouchBar/PikaTouchBar.swift
+++ b/Pika/TouchBar/PikaTouchBar.swift
@@ -75,10 +75,7 @@ class PikaTouchBarController: NSWindowController, NSTouchBarDelegate {
     }

     func createTouchBarColorButton(_ eyedropper: Eyedropper) -> NSButton? {
-        let action = eyedropper.type == .foreground
-            ? #selector(AppDelegate.triggerPickForeground)
-            : #selector(AppDelegate.triggerPickBackground)
-        let button = NSButton(title: "", target: nil, action: action)
+        let button = NSButton(title: "", target: nil, action: eyedropper.type.pickSelector)
         button.setButtonType(NSButton.ButtonType.toggle)
         eyedropper.$color.sink { color in
             self.updateButton(button: button, color: color)
```

If it's possible to update as well that'd be awesome. Thanks!